### ISSUE TITLE
List supported interpreters only once.

### ISSUE THREAD

**Issue by RJ722** [jendrikseipp/vulture#140]
We should only list the supported python interpreters once (currently they're listed in both
`.travis.yml` and `tox.ini`).

**Comment by The-Compiler** [comment]
`setup.py` (tags) and `README.md` (described in words) as well - I don't think there's much you can do to change that, though.

**Comment by jendrikseipp** [comment]
I agree. I'm not even sure whether we can remove the redundancy from tox.ini or .travis.yml, but Rahul will find that out for us :-)

**Comment by RJ722** [comment]
There is a tool named [`tox-travis`](https://github.com/tox-dev/tox-travis) which supports [environment detection](https://tox-travis.readthedocs.io/en/stable/envlist.html#advanced-configuration). We can try using it with Vulture, but given the workaround for running tests for py37 (#145) , I think it would rather complicate things. What do you guys think?

**Comment by The-Compiler** [comment]
I'm -1 on introducing any "magic" for things which get changed once all couple of months/years. It's likely to cause more trouble than it's worth :wink:

**Comment by jendrikseipp** [comment]
I agree with Florian. Probably best to leave things as their are.
